build.gradle

```diff
@@ -85,14 +85,16 @@ apply from: "${rootDir}/scripts/version-updater.gradle"
 
 ext {
 //    gliaSdkVersion = 'X.X.X' // this property is now attached dynamically during gradle sync
+  rxAndroid = '2.1.0'
+  rxJava = '2.2.6'
+  junitVersion = '4.13.2'
+  mockitoKotlinVersion = '5.2.1'
 
-  appCompatVersion = '1.3.1'
-  materialVersion = '1.6.1'
+  appCompatVersion = '1.6.1'
+  materialVersion = '1.11.0'
   constraintLayoutVersion = '2.1.4'
   navVersion = '2.3.5'
-  lifecycle_process = '2.3.1'
-  rxAndroid = '2.1.0'
-  rxJava = '2.2.6'
+  lifecycle_process = '2.7.0'
```

Comment:
Shall the whole `ext` section be also migrated to a version catalog approach?

Reply:
Yes, I will cleanup